etc/sync-function-authorization-module.js

```diff
@@ -0,0 +1,149 @@
+function() {
```

Comment:
The functions in this module are unchanged from when they were embedded directly in the sync function template file.